ste/sender-blockBlobFromURL.go

```diff
@@ -146,3 +147,14 @@ func (c *urlToBlockBlobCopier) generatePutBlockFromURL(id common.ChunkID, blockI
 		}
 	})
 }
+
+// GetDestinationLength gets the destination length.
+func (c *urlToBlockBlobCopier) GetDestinationLength() (int64, error) {
+	ctxWithLatestServiceVersion := context.WithValue(c.jptm.Context(), ServiceAPIVersionOverride, azblob.ServiceVersion)
```

Comment:
Do we need this special versioning for getting the length?  Just asking because we don't have it for some other blob types (e.g append) and I would imagine that getting the length is something that exist in all API versions... but maybe there's some reason I haven't thought of why we need it here.

Reply:
This can be future proofing for Azure Stack. They may support 2018-03 version later, which will allow users to import data into Azure Stack from public cloud. In that case, all calls should get the default version, and anything beyond 2018-03 would make them throw an error.

Reply:
I'm not sure I understand that @zezha-msft. Do you mean that the code, as it stands, should or shouldn't be changed?

Reply:
@JohnRusk we could keep this code, as Azure Stack might support 2018-03 one day, and cx might want to import data from public Azure into Azure Stack.